ui/fields/datetime.php

```diff
@@ -176,32 +173,38 @@ function podsCheckHtml5 () {
 		}
 
 		if ( ! podsCheckHtml5() ) {
-			args = altField( args, $element );
 			$element.val( '<?php echo esc_js( $formatted_value ); ?>' );
-			$element.<?php echo esc_js( $method ); ?>( args );
+			jQueryField();
 		}
 		<?php
 		} else {
 		?>
-		args = altField( args, $element );
-		$element.<?php echo esc_js( $method ); ?>( args );
+		jQueryField();
 		<?php
 		} //end if
 		?>
-		function altField( args, el ) {
-			var $el  = $( el ),
-				$alt = $el.clone();
+		function jQueryField() {
 
+			// Create alt field.
+			$alt = $element.clone();
 			$alt.attr( 'type', 'hidden' );
 			$alt.val( '<?php echo esc_attr( $mysql_value ) ?>' );
-			$el.after( $alt );
-
-			$el.attr( 'name', $el.attr( 'name' ) + '__ui' );
-			$el.attr( 'id', $el.attr( 'id' ) + '__ui' );
+			$element.after( $alt );
+			$element.attr( 'name', $element.attr( 'name' ) + '__ui' );
+			$element.attr( 'id', $element.attr( 'id' ) + '__ui' );
 
+			// Add alt field option.
 			args.altField = 'input#' + $alt.attr( 'id' );
+			// Fix manual user input changes.
+			args.onClose = function() {
+				$element.<?php echo esc_js( $method ); ?>( 'setDate', $element.val() );
```

Comment:
:no_entry_sign: Barf out, gag me with a spoon! esc_js()!This is for inline javascript, which is against our standards. Use json_encode instead.

<sub>hash: 1361f0ba35874dd5720cbf012a41e604</sub>